core/src/main/java/io/aiven/klaw/service/KafkaConnectControllerService.java

```diff
@@ -366,13 +367,12 @@ private List<KafkaConnectorModelResponse> getConnectorsPaginated(
                   new KafkaConnectorModelResponse();
               kafkaConnectorModelResponse.setSequence(counterInc);
 
-              List<String> envList = connectorSOT.getEnvironmentsList();
-              envList.sort(Comparator.comparingInt(orderOfEnvs::indexOf));
+              Set<String> set = new HashSet<>(connectorSOT.getEnvironmentsList());
 
               kafkaConnectorModelResponse.setConnectorId(connectorSOT.getConnectorId());
               kafkaConnectorModelResponse.setEnvironmentId(connectorSOT.getEnvironment());
               kafkaConnectorModelResponse.setEnvironmentsList(
-                  KlawResourceUtils.getConvertedEnvs(listAllEnvs, envList));
+                  KlawResourceUtils.getConvertedEnvs(listAllEnvs, set));
```

Comment:
Would you mind changing this to envSet or similar to make it more obvious whats in the set?

Reply:
the rest looks good

Reply:
makes sense
done